src/_assets/scss/inc/showcase.scss

```diff
@@ -0,0 +1,89 @@
+.showcase-grid {
+	margin-block: var(--gap);
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	gap: var(--gap);
+	border: var(--border-style);
+	border-radius: var(--radius-1);
+}
+
+.showcase-item {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	border: 1px solid var(--color-transparent-1);
+	border-radius: var(--radius-1);
+	overflow: hidden;
+	padding-bottom: .8rem;
+	header {
+		display: flex;
+		justify-content: space-between;
+		align-items: baseline;
+		padding: .8rem 1rem 0;
+		border-top: 1px solid var(--color-transparent-1);
+		svg {
+			position: relative;
+			top: 0.2em !important;
+			color: var(--color-transparent-2);
+		}
+	}
+	h2 {
```

Comment:
I usually don't use nesting in my styles (or only very sparingly), as it makes the source code hard to search based on the selector rendered in dev tools. But oh well, the whole docs site is a mess anyways already 😄